components/review/shared/reviewSplitPanelPage.tsx

```diff
@@ -34,6 +35,7 @@ export const ReviewSplitPanelPage: React.FC<Props> = ({
           {rightTitle ? <h2 className="text-xl">{rightTitle}</h2> : null}
           {rightContent}
         </div>
+        {children}
```

Comment:
Can you instead use `leftContent` and `rightContent`? Passing in children doesn't make it super clear where they'll end up for a split component like this.